components/indexer.py

```diff
@@ -1,30 +1,35 @@
 from magicbot import feedback
+import wpilib
 
 
 class Indexer:
     indexer_motors: list
     indexer_switches: list
+    ready_piston: wpilib.DigitalInput
```

Comment:
This is a horribly confusing name. Not only because it's not a piston, but also it returns `True` when... it's... not ready?